cyences_app_for_splunk/default/macros.conf

```diff
@@ -903,6 +907,10 @@ iseval = 0
 definition = index=o365
 iseval = 0
 
+[cs_o365_excluding_microsoft_teams_filter]
```

Comment:
Do not remove cs_o365_onedrive_or_sharepoint_link_accessed_by_external_user_filter macro. we are already using it. 

remove this one: cs_o365_excluding_microsoft_teams_filter